ansible/configs/rosa-consolidated/install_rosa_cli.yml

```diff
@@ -10,6 +10,9 @@
     src: /tmp/rosa-linux.tar.gz
     dest: /usr/local/bin/
     remote_src: true
+    owner: root
+    group: root
+    mode: '755'
```

Comment:
Recommend not using octal as it is error prone and hard to understand. Recommend symbolic notation `u=rwx,go=rx`